components/net/http_loader.rs

```diff
@@ -784,7 +784,7 @@ pub fn obtain_response<A>(request_factory: &HttpRequestFactory<R=A>,
         }
 
         if log_enabled!(log::LogLevel::Info) {
-            info!("{}", method);
+            info!("{} {}", method, connection_url);
```

Comment:
I was doing some debugging in XHR, and was having a hard time because the url wasn't being logged. Not really core business of this PR, but doing this in its own PR seems a bit much.